docs/root/start/sandboxes/opentelemetry.rst

```diff
@@ -119,3 +120,157 @@ In the ``Latency Samples`` of ``opentelemetry.proto.collector.trace.v1.TraceServ
+    opentelemetry-opentelemetry-1      |      -> peer.address: Str(172.30.0.7)
+    opentelemetry-opentelemetry-1      |      -> request_size: Str(0)
+    opentelemetry-opentelemetry-1      |      -> response_size: Str(37)
+    opentelemetry-opentelemetry-1      |      -> component: Str(proxy)
+    opentelemetry-opentelemetry-1      |      -> upstream_cluster: Str(envoy_cluster2)
+    opentelemetry-opentelemetry-1      |      -> upstream_cluster.name: Str(envoy_cluster2)
+    opentelemetry-opentelemetry-1      |      -> http.status_code: Str(200)
+    opentelemetry-opentelemetry-1      |      -> response_flags: Str(-)
+    opentelemetry-opentelemetry-1      |    {"kind": "exporter", "data_type": "traces", "name": "debug"}
+
+And the matching access log:
+
+.. code-block:: console
+    opentelemetry-opentelemetry-1      | 2024-05-02T15:08:01.867Z   info    LogsExporter {"kind": "exporter", "data_type": "logs", "name": "debug", "resource logs": 1, "log records": 1}
+    opentelemetry-opentelemetry-1      | 2024-05-02T15:08:01.867Z   info    ResourceLog #0
+    opentelemetry-opentelemetry-1      | Resource SchemaURL: 
+    opentelemetry-opentelemetry-1      | Resource attributes:
+    opentelemetry-opentelemetry-1      |      -> log_name: Str(otel_envoy_accesslog)
+    opentelemetry-opentelemetry-1      |      -> zone_name: Str()
+    opentelemetry-opentelemetry-1      |      -> cluster_name: Str()
+    opentelemetry-opentelemetry-1      |      -> node_name: Str()
+    opentelemetry-opentelemetry-1      | ScopeLogs #0
+    opentelemetry-opentelemetry-1      | ScopeLogs SchemaURL: 
+    opentelemetry-opentelemetry-1      | InstrumentationScope  
+    opentelemetry-opentelemetry-1      | LogRecord #0
+    opentelemetry-opentelemetry-1      | ObservedTimestamp: 1970-01-01 00:00:00 +0000 UTC
+    opentelemetry-opentelemetry-1      | Timestamp: 2024-05-02 15:08:01.188307 +0000 UTC
+    opentelemetry-opentelemetry-1      | SeverityText: 
+    opentelemetry-opentelemetry-1      | SeverityNumber: Unspecified(0)
+    opentelemetry-opentelemetry-1      | Body: Empty()
+    opentelemetry-opentelemetry-1      | Trace ID: 832518ad68578df8182cd76847693a93
+    opentelemetry-opentelemetry-1      | Span ID: 8af519328fd9ec0c
+    opentelemetry-opentelemetry-1      | Flags: 0
+    opentelemetry-opentelemetry-1      |    {"kind": "exporter", "data_type": "logs", "name": "debug"}
+
+Notice how the log record's `Trace ID: 832518ad68578df8182cd76847693a93` and `Span ID: 8af519328fd9ec0c` match the span's `Trace ID: 832518ad68578df8182cd76847693a93` and `ID: 8af519328fd9ec0c`.
+
+Step 6: How to replicate this in your setup
+*******************************************
+
+The configuration is divided in two parts: setting up the reporting of tracing data over OpenTelemetry, and the reporting of access logs:
+
+Set up a cluster to report to the OpenTelemetry collector (or another endpoint that accepts OTLP v1):
+
+.. code-block:: yaml
```

Comment:
this is failing when trying to build the rst

i _think_ the reason is that its confused by the yaml beneath - thinking its an rst directive/argument

i would suggest leaving a space after all `.. code-block:: ...` lines

in this particular case its an Envoy config - so it needs to go in a full bootstrap config and included - you can find similar examples with `git grep literalinclude docs/`

Reply:
(the rendered CI `/docs` will show for the last successful build and not at this commit)

Reply:
worth mentioning - you should be able to `literalinclude` the actual example configs 

in which case `git grep literalinclude docs/root/start/sandboxes` is probably more helpful

ill wait until the docs are rendering before reviewing again

Reply:
Thanks for the pointers, I gave it a much better try with 051d66012b